common/src/main/cpp/extensions/openxr_fb_spatial_entity_extension_wrapper.cpp

```diff
@@ -155,6 +161,51 @@ bool OpenXRFbSpatialEntityExtensionWrapper::_on_event_polled(const void *event)
 	return false;
 }
 
+bool OpenXRFbSpatialEntityExtensionWrapper::create_spatial_anchor(const Transform3D &p_transform, SpatialAnchorCreatedCallback p_callback, void *p_userdata) {
+	XrAsyncRequestIdFB request_id = 0;
+
+	Quaternion quat = Quaternion(p_transform.basis);
+	Vector3 pos = p_transform.origin;
+	XrPosef pose = {
+		{ quat.x, quat.y, quat.z, quat.w }, // orientation
+		{ pos.x, pos.y, pos.z }, // position
+	};
+
+	XrSpatialAnchorCreateInfoFB info = {
+		XR_TYPE_SPATIAL_ANCHOR_CREATE_INFO_FB, // type
+		nullptr, // next
+		reinterpret_cast<XrSpace>(get_openxr_api()->get_play_space()), // space
+		pose, // poseInSpace
+		get_openxr_api()->get_next_frame_time(), // time
```

Comment:
This should use `get_predicted_display_time` following the merging of https://github.com/godotengine/godot/pull/89734

Reply:
Yes! However, we'd need to update the `extension_api.json` we're using, and there's 4 other unrelated places where we're calling `get_next_frame_time()`. That feels like something for a dedicated PR.

Reply:
Sounds good! I've added https://github.com/GodotVR/godot_openxr_vendors/issues/140 to track.